cleverhans/model_zoo/deep_k_nearest_neighbors/dknn.py

```diff
@@ -16,6 +16,8 @@
 import matplotlib.pyplot as plt
 import numpy as np
 from six.moves import xrange
+import faiss
```

Comment:
Can you make the imports conditional on the corresponding flag value? (so users only have to install the library that they want to use)